src/components/Headline.js

```diff
@@ -10,8 +10,8 @@ function Headline() {
         <section className="headlineBg">
             <div className="flexBox column header">
                 <div className="innerContent">
-                    <h1 className="headline limited-title name"><span className="top-stagger">Ashley</span></h1>
-                    <h1 className="headline limited-title name"><span className="bottom-stagger">Bergsma</span></h1>
+                    <h1 className="headline limited-title"><span className="top-stagger">Ashley</span></h1>
```

Comment:
Name was a redundant class name that snuck in there... I was using it to control letter spacing and font size.... but that was being done in "limited-title"